parl/remote/client.py

```diff
@@ -243,20 +247,31 @@ def connect(master_address):
     assert len(master_address.split(":")) == 2, "please input address in " +\
         "{ip}:{port} format"
     global GLOBAL_CLIENT
+    cur_process_id = os.getpid()
     if GLOBAL_CLIENT is None:
-        GLOBAL_CLIENT = Client(master_address)
+        GLOBAL_CLIENT = Client(master_address, cur_process_id)
+    else:
+        if GLOBAL_CLIENT.process_id != cur_process_id:
+            GLOBAL_CLIENT = Client(master_address, cur_process_id)
 
 
 def get_global_client():
     """Get the global client.
 
+    If the process_id of the global client is not the same with the id of current process,
```

Comment:
To support process-based  programming, we will create a a global client in the new process.

Reply:
done